### ISSUE TITLE
Reset time out when there is mouse movement or input

### ISSUE THREAD

**Issue by ndgspn** [pelargir/auto-session-timeout#37]
I've got your gem up and running, but when I move the mouse, enter something still on the same page, does it still count as idle? because when I'm on the same page more than the specified timeout time, even when I'm typing something and moving the mouse, I still experience auto timeout and logged out automaticaly. Thank you!

**Comment by pelargir** [comment]
No, mouse movements or input don't reset the timeout. That's a great idea, though. I don't currently have bandwidth to implement something like this, but I'm always open to PRs from others.
